### ISSUE TITLE
Server: EmailService fails if MAILER_NOREPLY_EMAIL not set: Could not run maintenance: Error: Email is required

### ISSUE THREAD

**Issue by Artanicus** [laurent22/joplin#5669]
## Environment

Joplin version: 2.5.10-beta
Platform: Server
OS specifics: Tried with Docker image built from latest d22a29ee3d and the one labeled as `:2.5.10-beta`

Email config:
```
MAILER_ENABLED    = 1
MAILER_HOST       = "smtp.example.com"
MAILER_PORT       = 587
MAILER_SECURE     = 0
```

## Steps to reproduce

1. Trigger an email send, either by:
  - use the forgotten password feature with a valid account
  - As admin use "Send account confirmation email" from the user details
  - As admin, create a new user with no password set

## Describe what you expected to happen

An email is scheduled and during maintenance sent to the associated address.

## Logfile
```
2021-11-03 15:18:23: [error] EmailService: Could not run maintenance: Error: Email is required
    at EmailService.formatNameAndEmail (/home/joplin/packages/server/src/services/EmailService.ts:71:21)
    at EmailService.<anonymous> (/home/joplin/packages/server/src/services/EmailService.ts:92:17)
    at Generator.next (<anonymous>)
    at fulfilled (/home/joplin/packages/server/dist/services/EmailService.js:5:58)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

This blocks issue #5222 and probably should block full release of Server 2.5.10

**Comment by laurent22** [comment]
I think you need to define at a minimum MAILER_NOREPLY_EMAIL for the mailer to work. Could you try that? I'll add this to the checks when the server starts.

**Comment by laurent22** [comment]
Ok great, and this is now addressed in this commit: ce5c5d60420675a3703a551bd1f21aa333b6f8ed
